README.md

```diff
@@ -94,28 +93,28 @@ follows:
     gmk@Centos-7.img demo> cat hello
     world
 
-Once the developer has completed their environment the image file can be
-compressed and copied to any other system that has Singularity installed.
+Once, the developer has completed their environment the image file can
+be compressed and copied to any other system that has Singularity installed.
 If you do not have root on that system, you will not be able to make any
-changes to the image once on that system. But you will be able to use
-the container and access the data and files outside the container as
+changes to the image once on that system. But you will be able to use the
+container and access the data and files outside the container as
 easily as you would on your development system or virtual machine.
 
 ## Portability of Singularity container images
 Singularity images are highly portable between Linux distributions (as
 long as the binary format is the same). You can generate your image on
 Debian or CentOS, and run it on Mint or Slackware.
 
-Within a particular container one can include their programs, data,
+Within a particular container, one can include their programs, data,
 scripts and pipelines and thus portable to any other architecture
 compatible Linux system or distribution.
 
 ## Bootstrapping new images
-Generally when bootstrapping an image from scratch you must build it from
+Generally, when bootstrapping an image from scratch, you must build it from
 a compatible host. This is because you must use the distribution specific
-tools it comes with (e.g. Red Hat does not provide Debian's debootstrap).
-But once the image has been bootstrapped and includes the necessary bits
-to be self hosting (e.g. YUM on CentOS and apt-get on Debian/Ubuntu) then
+tools it comes with (e.g. Red Hat does not provide Debian's de-bootstrap).
```

Comment:
The official name from documentation is `debootstrap`, not `de-bootstrap` https://manpages.debian.org/stretch/debootstrap/debootstrap.8.en.html